demo/go/README.md

```diff
@@ -0,0 +1,71 @@
+# Rhino Go Demos
+
+This Go module contains demos for processing real-time audio (i.e. microphone) and audio files using the Rhino Speech-to-Intent engine.
+
+## Installation
+
+MicDemo uses [malgo](https://github.com/gen2brain/malgo) for cross-platform audio capture. It requires `cgo`, which on Windows may mean that you need to install a gcc compiler like [Mingw](http://mingw-w64.org/doku.php) to build it properly. 
+
+## Usage
+
+NOTE: The working directory for the following go commands is:
+
+```console
+rhino/demo/go
```

Comment:
nitpick but add "cd" since this is what they're supposed to type in their prompt